### PR TITLE
Complete mobile version Pull Request

### PR DESCRIPTION
**Pull request by federicaulzurrun**
In this project, I added to my portfolio page two more sections. 1st the works sections, where you can see projects that I've already done. In the second part, I incorporate the about me section, where you can see which languages I use while I'm programming!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,13 +29,140 @@ <h1 class="title">I'm Federica Glad to see you! </h1>
...
+                </div>
+                <p class= "primary-text">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                <ul class="tags-block">
+                    <li class="list-item"><button disabled class= "btn btn-tags"> html</button></li>
+                    <li class="list-item"><button disabled class= "btn btn-tags"> css</button></li>
+                    <li class="list-item"><button disabled class= " btn btn-tags"> javaScript</button></li>
+                </ul>
+                <div class="action">
+                    <button disabled class="btn normal-b"> See project</button>
+                </div>
```
Comment (Owusu-Desmond) [inline index.html:110]: - [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
image

File: index.html [hunk 2]
```diff
@@ -29,13 +29,140 @@ <h1 class="title">I'm Federica Glad to see you! </h1>
...
+                <p class= "primary-text multi">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                <ul class="tags-block">
+                    <li class="list-item"><button disabled class= "btn btn-tags"> html</button></li>
+                    <li class="list-item"><button disabled class= "btn btn-tags"> css</button></li>
+                    <li class="list-item"><button disabled class= "btn btn-tags"> javaScript</button></li>
+                </ul>
+                <div class="action">
+                    <button disabled class="btn normal-b"> See project</button>
+                </div>
```
Comment (Owusu-Desmond) [inline index.html:134]: - [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
image

File: index.html [hunk 3]
```diff
@@ -29,13 +29,140 @@ <h1 class="title">I'm Federica Glad to see you! </h1>
...
+            <ul class="networks-about">
+                <li><a href=""><img src="img/GitHub.png" alt="20px"></a></li>
+                <li ><a href=""><img src="img/Linkedin icon.png" alt=""></a></li>
+                <li><a href=""><img src="img/Peace.png" alt="20px"></a></li>
+                <li><a href=""><img src="img/Twitter.png" alt="20px"></a></li>
+                <li><a href=""><img src="img/M.png" alt="20px"></a></li>
+            </ul>
+            <div class="about-action">
+                <button disabled class="btn normal-b">Get my resume</button>
+            </div>
```
Comment (Owusu-Desmond) [inline index.html:150]: - [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
image

File: index.html [hunk 4]
```diff
@@ -29,13 +29,140 @@ <h1 class="title">I'm Federica Glad to see you! </h1>
...
+                </div>
+                <p class= "primary-text multi">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                <ul class="tags-block">
+                    <li class="list-item"><button disabled class= "btn btn-tags"> html</button></li>
+                    <li class="list-item"><button disabled class= "btn btn-tags"> css</button></li>
+                    <li class="list-item"><button disabled class= " btn btn-tags"> javaScript</button></li>
+                </ul>
+                <div class="action">
+                    <button disabled class="btn normal-b"> See project</button>
+                </div>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
image

File: index.html [hunk 5]
```diff
@@ -29,13 +29,140 @@ <h1 class="title">I'm Federica Glad to see you! </h1>
...
+                <p class= "primary-text">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                <ul class="tags-block">
+                    <li class="list-item"><button disabled class= "btn btn-tags"> html</button></li>
+                    <li class="list-item"><button disabled class= "btn btn-tags"> css</button></li>
+                    <li class="list-item"><button disabled class= " btn btn-tags"> javaScript</button></li>
+                </ul>
+                <div class="action">
+                    <button disabled class="btn normal-b"> See project</button>
+                </div>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @federicaulzurrun , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Your UI looks Nice✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by federicaulzurrun** [comment]
@Owusu-Desmond i should do the 3 options? I don't understand what I'm supposed to do because the button should be disabled, so the design is not going to be the same

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @federicaulzurrun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] Pull request has a descriptive title and summary :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] GitHub flow is correctly used :white_check_mark: 

## Optional suggestions

- [ Optional ] Good job implementing the different button states, however, please enable all the disabled buttons, this is because in your portfolio you'll need all buttons and links to be clickable when a user is interacting with them.

- [ Optional ] I've noticed that the license link on your readme file is not working, it leads to a 404 page not found error. I would like to suggest that you fix this by adding `License` file in your root directory and provide a link to the license on the readme file. You can use the contents of the  `MIT` license on this link [Click Here >>](https://choosealicense.com/licenses/mit/)

Screenshot from 2023-02-19 00-59-42

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**